src/storage/StorageConfig.ts

```diff
@@ -66,74 +65,75 @@ export class StorageConfig {
         return instance
     }
 
-    async _poll(pollInterval: number) {
+    private async _poll(pollInterval: number): Promise<void> {
+        if (this._stopPoller) { return }
```

Comment:
Not currently possible to hit this but this check removes any possibility that this can accidentally be triggered after cleanup is called.